.github/workflows/push_pr.yml

```diff
@@ -1,10 +1,11 @@
-name: Push/PR pipeline
```

Comment:
this might be missing , Last Pr to docker save me some time :) 
https://github.com/newrelic/nri-docker/blob/master/.github/workflows/push_pr.yml#L23

Reply:
done, I added it to the checklist